template_package/test/node_test.cpp

```diff
@@ -0,0 +1,39 @@
+#include <gtest/gtest.h>
```

Comment:
Should we include a copyright statement here?

Reply:
Added